### PR TITLE
[ign-gazebo2] Update BitBucket links

### PR DESCRIPTION
**Pull request by chapulina**
Signed-off-by: Louise Poubel <louise@openrobotics.org>

### CODE REVIEW

**Comment by chapulina** [comment]
@osrf-jenkins run tests

File: README.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ Follow these steps to run tests and static code analysis in your clone of this r
     make codecheck
     ```
 
-See the [Writing Tests section of the contributor guide](https://bitbucket.org/ignitionrobotics/ign-gazebo/src/406665896aa40bb42f14cf61d48b3d94f2fc5dd8/CONTRIBUTING.md#markdown-header-writing-tests) for help creating or modifying tests.
+See the [Writing Tests section of the contributor guide](https://github.com/ignitionrobotics/ign-gazebo/blob/master/CONTRIBUTING.md#writing-tests) for help creating or modifying tests.
```
Comment (maryaB-osr) [inline README.md:262]:
```suggestion
See the [Writing Tests section of the contributor guide](https://ignitionrobotics.org/docs/all/contributing#writing-tests) for help creating or modifying tests.
```

File: docker/scripts/install_common_deps.sh [hunk 2]
```diff
@@ -18,7 +18,7 @@ sudo apt-get install -y \
   curl \
   git \
   g++-8 \
-  mercurial \
+  git \
```
Comment (maryaB-osr) [inline docker/scripts/install_common_deps.sh:21]:
```suggestion
```

File: tutorials/levels.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ being added and removed.
 Take a look at the 2D example below. This example focuses on a single performer,
 but the same logic can be extended to multiple performers.
 
-<img src="https://bytebucket.org/ignitionrobotics/ign-gazebo/raw/9e88ca13a2b3b51a1b39abeb2b5b8db23f2e74e3/doc/architecture_design/01.png"/>
+<img src="https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/master/doc/architecture_design/01.png"/>
```
Comment (maryaB-osr) [inline tutorials/levels.md:81]:
```suggestion
<img src="https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/d62080da95edbb172c47eac883ec4b707b59bb38/doc/architecture_design/01.png"/>
```

File: tutorials/levels.md [hunk 4]
```diff
@@ -116,23 +116,23 @@ Let's take a look at how levels are loaded / unloaded as the performer moves:
     * `M1` and `M3`, because they belong to the level.
     * `M6`, because it is global.
 
-    <img src="https://bytebucket.org/ignitionrobotics/ign-gazebo/raw/9e88ca13a2b3b51a1b39abeb2b5b8db23f2e74e3/doc/architecture_design/02.png"/>
+    <img src="https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/raw/9e88ca13a2b3b51a1b39abeb2b5b8db23f2e74e3/doc/architecture_design/02.png"/>
```
Comment (maryaB-osr) [inline tutorials/levels.md:119]:
```suggestion
    <img src="https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/d62080da95edbb172c47eac883ec4b707b59bb38/doc/architecture_design/02.png"/>
```

File: tutorials/levels.md [hunk 5]
```diff
@@ -116,23 +116,23 @@ Let's take a look at how levels are loaded / unloaded as the performer moves:
...
 
 2. The performer moves south towards `L3` and enters its buffer zone, triggering
     a load of that level's models, `M4` and `M5`. Note that at this moment, both
     `L1` and `L3` are loaded.
 
-    <img src="https://bytebucket.org/ignitionrobotics/ign-gazebo/raw/9e88ca13a2b3b51a1b39abeb2b5b8db23f2e74e3/doc/architecture_design/03.png"/>
+    <img src="https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/raw/9e88ca13a2b3b51a1b39abeb2b5b8db23f2e74e3/doc/architecture_design/03.png"/>
```
Comment (maryaB-osr) [inline tutorials/levels.md:125]:
```suggestion
    <img src="https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/d62080da95edbb172c47eac883ec4b707b59bb38/doc/architecture_design/03.png"/>
```

File: tutorials/levels.md [hunk 6]
```diff
@@ -116,23 +116,23 @@ Let's take a look at how levels are loaded / unloaded as the performer moves:
...
 
 3. The performer moves further south, exiting `L1` and entering `L3`. However,
   `L1` is still loaded, since `R1` is still within its buffer zone.
 
-    <img src="https://bytebucket.org/ignitionrobotics/ign-gazebo/raw/9e88ca13a2b3b51a1b39abeb2b5b8db23f2e74e3/doc/architecture_design/04.png"/>
+    <img src="https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/raw/9e88ca13a2b3b51a1b39abeb2b5b8db23f2e74e3/doc/architecture_design/04.png"/>
```
Comment (maryaB-osr) [inline tutorials/levels.md:130]:
```suggestion
    <img src="https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/d62080da95edbb172c47eac883ec4b707b59bb38/doc/architecture_design/04.png"/>
```

File: tutorials/levels.md [hunk 7]
```diff
@@ -116,23 +116,23 @@ Let's take a look at how levels are loaded / unloaded as the performer moves:
...
 
 4. Eventually `R1` moves beyond `L1`'s buffer, triggering an unload of `L1`. The
   main effect is unloading `M1`.
 
-    <img src="https://bytebucket.org/ignitionrobotics/ign-gazebo/raw/9e88ca13a2b3b51a1b39abeb2b5b8db23f2e74e3/doc/architecture_design/05.png"/>
+    <img src="https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/raw/9e88ca13a2b3b51a1b39abeb2b5b8db23f2e74e3/doc/architecture_design/05.png"/>
```
Comment (maryaB-osr) [inline tutorials/levels.md:135]:
```suggestion
    <img src="https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/d62080da95edbb172c47eac883ec4b707b59bb38/doc/architecture_design/05.png"/>
```

File: tutorials/levels.md [hunk 8]
```diff
@@ -247,7 +247,7 @@ ign service -s /world/levels/level/set_performer --reqtype ignition.msgs.StringM
 The following is a world file that could be an instance of the world shown in
 the figure
 
-<img src="https://bytebucket.org/ignitionrobotics/ign-gazebo/raw/9e88ca13a2b3b51a1b39abeb2b5b8db23f2e74e3/doc/architecture_design/06.png"/>
+<img src="https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/raw/9e88ca13a2b3b51a1b39abeb2b5b8db23f2e74e3/doc/architecture_design/06.png"/>
```
Comment (maryaB-osr) [inline tutorials/levels.md:250]:
```suggestion
<img src="https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/d62080da95edbb172c47eac883ec4b707b59bb38/doc/architecture_design/06.png"/>
```

File: tutorials/detachable_joints.md [hunk 9]
```diff
@@ -8,7 +8,7 @@ kinematic topology has to be a tree, i.e., kinematic loops are not currently
 supported. This affects the choice of the parent link, and therefore, the
 parent model, which is the model that contains the `DetachableJoint` system.
 
-For example, [detachable_joint.sdf](https://bitbucket.org/ignitionrobotics/ign-gazebo/src/default/examples/worlds/detachable_joint.sdf)
+For example, [detachable_joint.sdf](https://github.com/ignitionrobotics/ign-gazebo/blob/master/examples/worlds/detachable_joint.sdf)
```
Comment (maryaB-osr) [inline tutorials/detachable_joints.md:11]:
```suggestion
For example, [detachable_joint.sdf](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo2/examples/worlds/detachable_joint.sdf)
```

**Pull request merged**